includes/class-swedbank-pay-instant-capture.php

```diff
@@ -36,15 +36,15 @@ class Swedbank_Pay_Instant_Capture {
 	 * Constructor.
 	 */
 	public function __construct() {
-		add_action( 'woocommerce_order_status_on-hold', array( $this, 'maybe_capture_instantly' ), 50, 10 );
+		add_action( 'woocommerce_order_status_completed', array( $this, 'maybe_capture_instantly' ), 50, 10 );
```

Comment:
Question: Is the status for this action correct? Isn't `on-hold` previously what we changed to be `processing` now with the new logic? If the order is set to `completed`, for example if the order is only for digital and downloadable products, wont the order management already handle this on with the [woocommerce_order_status_changed](https://github.com/krokedil/swedbank-pay-woocommerce-paymentmenu/blob/main/includes/class-swedbank-pay-admin.php#L50) action?